draft-tigress-requirements.md

```diff
@@ -144,14 +144,15 @@ General terms:
 - (Req-NontechnicalUX) Solution SHALL enable secure credential transfer for non technical users.
 - (Req-SmoothUX) Solution SHALL allow for user experience where neither Sender nor Receiver is presented with raw data required only by the secure transfer protocol. The data SHOULD only be parsed programmatically and not required to be presented to the end user. This data SHOULD never be visible to said user in whichever messaging application the sender chose to initiate the transfer on. This eliminates the possibility of merely sending the requisite data inline, through an SMS or email for example, rather than leveraging an Intermediary server.
 - (Req-Connectivity) Sender and Receiver SHALL be allowed to be online at different times. Sender and Receiver SHALL never need to be online at the same time.
-- (Req-init) Solution SHOULD allow Sender to initiate credential transfer to Receiver over any messaging channel, with various degrees of security.
-- (Req-P2P) A credential transfer SHALL be strictly from one device to another (group sharing is not a goal).
+- (Req-UrlLink) Solution SHALL allow to share credentials as a UrlLink to provisioning information stored on intermediary server. The link SHALL easily be shared between Sender and Receiver device over various communication channels.
```

Comment:
I followed up in https://github.com/dimmyvi/tigress-requirements/issues/11 about this. I don't think we should add it.